payroll/context_processors.py

```diff
@@ -9,8 +10,14 @@ def inspiration_slugs(request):
     except:
         chicago_slug = None
 
+    years = StandardizedFile.objects\
+        .distinct('reporting_year')\
+        .values('reporting_year')
+    reversed_years = reversed(years)
```

Comment:
FYI – you can use [`order_by`](https://docs.djangoproject.com/en/3.0/ref/models/querysets/#order-by) to do this in your ORM query.